_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
+layout: article
+section: On-prem
+category: component
+order: 1
+since: 20190119
+---
+
+One of the first problems this article solves is simplifying how users configure the applications they deploy to Kubernetes clusters.
+
+With that in mind, we are representing a detailed step by step instruction on how to implement the {{site.data.tenant.name}} platform's cluster and all the required Kubernetes descriptors onto a customer's hardware utilizing KSonnet.
```

Comment:
You don't implement a cluster but you deploy {{site.data.tenant.name}} platform into a running Kubernetes cluster.